scripts/validate_packaged_v8_resources.py

```
@@ -0,0 +1,144 @@
+) -> None:
+    """Validate exact package-local inventory and exercise every resource loader."""
+
+    site_packages = site_packages.resolve(strict=True)
+    runtime_root = runtime_root.resolve(strict=True)
+    package_root = Path(str(resources.files("defenseclaw"))).resolve(strict=True)
+    canonical_package_path = site_packages / "defenseclaw"
+    is_junction = getattr(canonical_package_path, "is_junction", None)
+    if canonical_package_path.is_symlink() or (is_junction is not None and is_junction()):
+        raise PackagedV8ResourceError(
+            f"{label} DefenseClaw package root uses symlink or junction indirection: {canonical_package_path}"
+        )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)